llap-common/src/protobuf/LlapDaemonProtocol.proto

```diff
@@ -223,4 +231,5 @@ service LlapDaemonProtocol {
 service LlapManagementProtocol {
   rpc getDelegationToken(GetTokenRequestProto) returns (GetTokenResponseProto);
   rpc purgeCache(PurgeCacheRequestProto) returns (PurgeCacheResponseProto);
+  rpc getLoadMetrics(GetLoadMetricsRequestProto) returns (GetLoadMetricsResponseProto);
```

Comment:
Would rename from getLoadMetrics to getDaemonMetrics

Reply:
Done